rc_int

```diff
@@ -0,0 +1,19 @@
+export APACHE_BASE_PATH=main
+export API_URL=//mf-chsdi3.int.bgdi.ch
+export DBHOST=pg.bgdi.ch
```

Comment:
Yes, it's currently wrong

pg.bgdi.ch should be limited to production systems only. This is to avoid that in some testing/integration environment a run-wild tests consumes all our connections there.

All other system should use pg-sandbox (or pg-0.dev)